druid-shell/src/backend/wayland/application.rs

```diff
@@ -292,6 +300,9 @@ impl Application {
                         }
                         // Dont worry if they go away - we will just stop receiving events. If the
                         // capability comes back we will start getting events again.
+
+                        // TODO: Acording to the protocol the above comment is not true, and describes compositor bug,
```

Comment:
just change the comment if its wrong. ;)

Reply:
Right, it was a quick sketchy comment, will update it and add a reference while at it.